diff --git a/dist/api/environment.js b/dist/api/environment.js
index b749a4dc8bd7d17b4c35d9275d6a9cf9f9a74ef0..c2a1720a397a54fecbf5ddeb49e404561f2cc784 100644
--- a/dist/api/environment.js
+++ b/dist/api/environment.js
@@ -13,7 +13,7 @@ class Environment {
         request.setHeaderParam("idempotency-key", options === null || options === void 0 ? void 0 : options.idempotencyKey);
         return request.send(this.requestCtx, environmentOut_1.EnvironmentOutSerializer._fromJsonObject);
     }
-    import(environmentIn, options) {
+    ["import"](environmentIn, options) {
         const request = new request_1.SvixRequest(request_1.HttpMethod.POST, "/api/v1/environment/import");
         request.setHeaderParam("idempotency-key", options === null || options === void 0 ? void 0 : options.idempotencyKey);
         request.setBody(environmentIn_1.EnvironmentInSerializer._toJsonObject(environmentIn));
